frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -357,6 +362,37 @@ export function ComponentCompatibility() {
   );
 }
 
+// Card visualizations
+
+// @ts-expect-error: incompatible prop types with registerVisualization
+registerVisualization(ObjectDetail);
+
+export const CardVisualizationsLightTheme = Template.bind({});
+CardVisualizationsLightTheme.args = {
+  ...defaultArgs,
+  dashboard: createDashboard({
+    dashcards: [
+      createMockDashboardCard({
+        id: DASHCARD_TABLE_ID,
+        dashboard_tab_id: TAB_ID,
+        card: createMockCard({
+          id: CARD_TABLE_ID,
+          name: "Table detail",
+          display: "object",
```

Comment:
I pretty much only change the display to `object` the rest is copied over from a table card.